Formula/tuist@3.22.0.rb

```diff
@@ -0,0 +1,40 @@
+class TuistAT3220 < Formula
+  desc "Create, maintain, and interact with Xcode projects at scale"
+  homepage "https://tuist.io"
+  url "https://github.com/tuist/tuist/archive/refs/tags/3.22.0.tar.gz"
+  sha256 "e719d4c47990f79e821392ecf16e5e313eba5d2ccefecd7a602f4eb51e5659ac"
+  license "MIT"
+  head "https://github.com/tuist/tuist.git", branch: "main"
+
+  # https://github.com/tuist/tuist/blob/3.22.0/.xcode-version#L1
+  depends_on xcode: ["14.3", :build]
+
+  # https://github.com/tuist/tuist/blob/3.22.0/Package.swift#L14
+  depends_on macos: :monterey
+
+  def install
+    # https://github.com/tuist/tuist/blob/3.22.0/.github/workflows/tuist.yml#L81
+    system "swift", "build",
+      "--configuration", "release",
+      "--product", "tuist",
+      "--disable-sandbox"
+
+    # https://github.com/tuist/tuist/blob/3.22.0/.github/workflows/tuist.yml#L85
+    system "swift", "build",
+      "--configuration", "release",
+      "--product", "ProjectDescription",
+      "--disable-sandbox"
+
+    # https://github.com/tuist/tuist/blob/3.22.0/Sources/TuistEnvKit/Installer/BuildCopier.swift#L13
+    bin.install ".build/release/tuist"
+    bin.install "projects/tuist/vendor"
```

Comment:
The Templates directory is missing here.

https://github.com/vanyauhalin/homebrew-tuist/blob/dc05e3df94a10d2c566af5eee6444788b60e6dd7/Formula/tuist%403.21.1.rb#L29-L34